### DISCUSSION TITLE
Are these routes safe?

### DISCUSSION THREAD

**Discussion by Endy-c** [facade/ignition#454] [Q&A]
It has routes like `POST _ignition/execute-solution`
and I don't find any configuration to disable it, is that safe?

**Comment by freekmurze** [comment]
You can disable it here: https://github.com/spatie/laravel-ignition/blob/6e1d0089d8c2b9b3a36bde7140097e818666c1de/config/ignition.php#L128-L141

Those routes will only work when app debug is set to `true`.

**Reply by Endy-c** [reply]
Thanks!

**Reply by Endy-c** [reply]
Because these routes didn't exist with command `php artisan route:list` in Laravel 8, so I concern about it when they showed up in Laravel 9.
